conf/jenkins/config.xml

```diff
@@ -22,8 +22,8 @@
       <master>zk://leader.mesos:2181/mesos</master>
       <description></description>
       <frameworkName>jenkins</frameworkName>
-      <role>*</role>
-      <slavesUser>root</slavesUser>
+      <slavesUser></slavesUser>
```

Comment:
I removed them since both are now configurable/guaranteed to be set when the package is installed- I guess it doesn't matter though.